python/grass/jupyter/utils.py

```diff
@@ -203,7 +203,7 @@ def query_vector(coord, vector_list, distance):
     """
     output_list = ["""<table>"""]
 
-    for vector in vector_list:
+    for vector in vector_list:  # pylint: disable=too-many-nested-blocks
```

Comment:
Have you considered actually addressing this instead of disabling it? It think it would help readability of the code.